spec/index.html

```diff
@@ -217,8 +217,8 @@
             publisher: "The Open Source Geospatial Foundation"
           },
           "GeoServer": {
-            title: "GeoServer MapML Community Module",
-            href: "https://docs.geoserver.org/latest/en/user/community/mapml/index.html",
+            title: "GeoServer MapML Extension",
```

Comment:
Should it rather say <q>GeoServer MapML Extension Module</q>?

Reply:
It's good as is.